crates/uv-virtualenv/src/virtualenv.rs

```diff
@@ -143,21 +147,60 @@ pub(crate) fn create(
     // Create a `.gitignore` file to ignore all files in the venv.
     fs::write(location.join(".gitignore"), "*")?;
 
+    let executable_target = if upgradeable && interpreter.is_standalone() {
+        if let Some(minor_version_link) = PythonMinorVersionLink::from_executable(
+            base_python.as_path(),
+            &interpreter.key(),
+            preview,
+        ) {
+            if tracing::enabled!(tracing::Level::DEBUG) {
```

Comment:
I think it's okay not to gate with this, we don't tend to do that unless we're performing a particularly expensive operation.

Reply:
Updated in #14029